src/main/java/com/uid2/client/Uid2Encryption.java

```diff
@@ -43,13 +43,19 @@ static DecryptionResponse decrypt(String token, KeyContainer keys, Instant now,
         }
         else if (unsignedByte  == AdvertisingTokenVersion.V4.value())
         {
-            //same as V3 but use Base64URL encoding
-            return decryptV3(Uid2Base64UrlCoder.decode(token), keys, now, identityScope, domainName, clientType, 4);
+            // Accept either base64 or base64url encoding.
+            return decryptV3(Base64.getDecoder().decode(base64UrlToBase64(token)), keys, now, identityScope, domainName, clientType, 4);
```

Comment:
do we still need Uid2Base64UrlCoder class/methods?

Reply:
We still use it to encode, and the decoding is used on the first four bytes of the token. I could have changed that, but this way [it matches the .NET SDK](https://github.com/IABTechLab/uid2-client-net/blob/e0b5899a115942f9b0d5ffd0340aa882285aa9cb/src/UID2.Client/UID2Encryption.cs#L38).